jdk-tool-plugin/src/main/java/hudson/tools/JDKInstaller.java

```diff
@@ -275,7 +283,7 @@ I tried to locate exactly how InstallShield parses the arguments (and why it use
                 out.println(Messages.JDKInstaller_FailedToInstallJDK(r));
                 // log file is in UTF-16
                 try (InputStreamReader in = new InputStreamReader(fs.read(logFile), "UTF-16")) {
-                    IOUtils.copy(in, new OutputStreamWriter(out));
+                    IOUtils.copy(in, new OutputStreamWriter(out, Charset.defaultCharset()));
```

Comment:
`defaultCharset` on the master need not match the default charset on the agent.

Reply:
Ok, should I just pick some standard charset?

Reply:
Do not know what is appropriate for this case. It is possible to obtain the agent’s charset if that is required (currently there is not a standard method to do so).